src/js/base/Container.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export function Container({ children }) {
+    return <div className="max-w-full px-[35px] pl-[98px] w-screen">{children}</div>;
```

Comment:
Don't use these exact size classes. We want to use the built-ins like `px-1`, `px-2` etc. Find the values that work. It is OK if it changes slightly.